core/src/main/scala/kafka/log/LocalLog.scala

```diff
@@ -946,9 +947,19 @@ object LocalLog extends Logging {
       info(s"${logPrefix}Deleting segment files ${segmentsToDelete.mkString(",")}")
```

Comment:
this log will also print a lot elastic log inner state. but not changed in this pr